content/ja/docs/concepts/workloads/controllers/job.md

```diff
@@ -0,0 +1,603 @@
+[フィーチャーゲート](/ja/docs/reference/command-line-tools-reference/feature-gates/)を有効にする必要があります。
+
+有効にした場合、コントロールプレーンは下記に示す機能で新しいJobを追跡します。この機能が有効になる前に作成されたJobは影響を受けません。ユーザーとして、唯一に実感できる違いは、コントロールプレーンのJob完了ステータス追跡はより正確になったということです。
+{{< /note >}}
+
+この機能が有効でない場合、Job {{< glossary_tooltip term_id="controller" >}}はクラスター内に存在するPodを数えてJobステータスを追跡します。つまり`succeeded`Podと`failed`Podのカウンタを保持します。
+しかし、Podは以下のように削除されることもあります:
+- Nodeがダウンしたときに、孤立した(Orphan)Podはガベージコレクターにより削除されます。
+- 閾値に達すると、終了したPod(`Succeeded`または`Failed`フェーズを問わず)はガベージコレクター二より削除されます。 
+- 人為的にJobに属するPodを削除します。
+- 外部コントローラー(Kubernetesの一部として提供されていない)よりPodを削除したり置き換えたりします。
+
+クラスターで`JobTrackingWithFinalizers`機能を有効にすると、コントロールプレーンは任意のJobに属するPodを追跡し、そのようなPodがAPIサーバーから削除された場合に通知します。そのために、Jobコントローラーにより作成されたPodは`batch.kubernetes.io/job-tracking`Finalizerを持っています。コントローラーはPodがJobステータスに計上された後にのみFinalizerを削除し、他のコントローラーやユーザーによるPodの削除を可能にします。
+
+Jobコントローラーは、新しいJobに対してのみ新しいアルゴリズムを使用します。この機能が有効になる前に作成されたJobは影響を受けません。JobコントローラーがPod FinalizerでJob追跡しているかどうかは、Jobが`batch.kubernetes.io/job-tracking`というアノテーションを持っているかどうかで判断できます。
+このアノテーションを手動で追加または削除しては**いけません**。
+
+## 代替案  {#alternatives}
+
+### 単なるPod  {#bare-pods}
+
+Podが動作しているノードが再起動または故障した場合、Podは終了し、再起動されません。しかし、終了したPodを置き換えるため、Jobが新しいPodを作成します。このため、たとえアプリケーションが1つのPodしか必要としない場合でも、単なるPodではなくJobを使用することをお勧めします。 
+
+### Replication Controller  {#replication-controller}
+
+Jobsは[Replication Controllers](/docs/concepts/workloads/controllers/replicationcontroller/)を補完するものです。
+Replication Controllerは、終了することが想定されていないPod(Webサーバーなど)を管理し、Jobは終了することが想定されているPod(バッチタスクなど)を管理します。
+
+[Podのライフサイクル](/ja/docs/concepts/workloads/pods/pod-lifecycle/)で説明したように、`Job`は`RestartPolicy`が`OnFailure`か`Never`と設定されているPodに*のみ*適用されます。(注意:`RestartPolicy`が設定されていない場合、デフォルト値は`Always`になります)
+
+
+### シングルJobによるコントローラーPodの起動  {#single-job-starts-controller-pod}
+
+もう一つのパターンは、一つのJobが一つPodを作り、そのPodがカスタムコントローラーのような役割を果たし、他のPodを作ります。これは最も柔軟性がありますが、使い始めるにはやや複雑で、Kubernetesとの統合もあまりできません。
+
+このパターンの一例としては、Sparkマスターコントローラーを起動するスクリプトがあり、そのスクリプトを実行するPodをJobで起動し、([spark事例](https://github.com/kubernetes/examples/tree/master/staging/spark/README.md)を参照)、sparkドライバーを実行してクリーンアップする、ということが挙げられます。
+
+
+この方法のメリットは、Jobオブジェクトで実行されているので、プロセスが完了する保障があると同時に、どのPodを作成し、どのように仕事を割り当てるかを完全に制御できることです。
+
+## {{% heading "whatsnext" %}}
+
+* [Pods](/ja/docs/concepts/workloads/pods/)について学ぶ。
+* Jobのさまざまな実行方法について学ぶ:
+   * [ワークキューを用いた粒度の粗い並列処理](/docs/tasks/job/coarse-parallel-processing-work-queue/)
+   * [ワークキューを用いた粒度の粗い並列処理](/docs/tasks/job/fine-parallel-processing-work-queue/)
+   * [静的な処理の割り当てを使用した並列処理のためのインデックス付きJob](/ja/docs/tasks/job/indexed-parallel-processing-static/) を使う(beta段階)
+   * テンプレートを元に複数のJobを作成: [拡張機能を用いた並列処理](/docs/tasks/job/parallel-processing-expansion/)
+* [終了したJobの自動クリーンアップ](#clean-up-finished-jobs-automatically)のリンクから、クラスターが完了または失敗したJobをどのようにクリーンアップするかをご確認ください。
+* `Job`はKubernetes REST APIの一部です。Job APIを理解するために、{{< api-reference page="workload-resources/job-v1" >}}オブジェクトの定義をお読みください。
```

Comment:
```suggestion
* `Job`はKubernetes REST APIの一部です。JobのAPIを理解するために、{{< api-reference page="workload-resources/job-v1" >}}オブジェクトの定義をお読みください。
```

Reply:
I am not sure that word of "Job API" exists as proper noun, that's why I left comment here.